src/rpa_im_time.F

```diff
@@ -738,23 +734,24 @@ SUBROUTINE compute_mat_dm_global(fm_scaled_dm_occ_tau, fm_scaled_dm_virt_tau, ta
                                                             ncol_local, nrow_local, size_dm_occ, &
                                                             size_dm_virt
       INTEGER, DIMENSION(:), POINTER                     :: col_indices, row_indices
-      REAL(KIND=dp)                                      :: tau
+      REAL(KIND=dp)                                      :: stabilize_exp, tau
 
       CALL timeset(routineN, handle)
 
       IF (memory_info .AND. unit_nr > 0) WRITE (UNIT=unit_nr, FMT="(T3,A,T75,i6)") &
          "RPA_LOW_SCALING_INFO| Started with time point: ", jquad
 
       tau = tau_tj(jquad)
+      stabilize_exp = 70.0_dp
```

Comment:
The same like with num_points_per_magnitude:
`REAL(KIND=dp), PARAMETER :: stabilize_exp = 70.0_dp`